integration-libs/punchout/root/facade/punchout.facade.ts

```diff
@@ -33,7 +34,8 @@ export function punchoutFacadeFactory() {
 })
 export abstract class PunchoutFacade {
   /**
-   * Abstract method used to get Punchout Session
+   * Abstract method used to get Punchout Session, worflow:
+   * occ api request, logout previous user, login punchout user, load cart
    * @param sessionId is the sesssion Id given by ARIBA via url param
    */
   abstract getPunchoutSession(
```

Comment:
planning to rename as 'initPunchoutSession' as it does much more than requesting server punchout session.

Reply:
renaming tasks covered in 
CXSPA-9873 | Improve source code naming